src/NBitcoin/ConsensusOptions.cs

```diff
@@ -52,6 +55,7 @@ public ConsensusOptions()
             this.MaxStandardTxWeight = 400000;
             this.MaxBlockBaseSize = 1000000;
             this.MaxBlockSigopsCost = 80000;
+            this.MaxStandardTxSigopsCost = this.MaxBlockSigopsCost / 5;
```

Comment:
This is sort of a default value?

Reply:
on bitcoin yes, it is
https://github.com/bitcoin/bitcoin/blob/78dae8caccd82cfbfd76557f1fb7d7557c7b5edb/src/policy/policy.h#L30

Reply:
`static const unsigned int MAX_STANDARD_TX_SIGOPS_COST = MAX_BLOCK_SIGOPS_COST/5;`

Reply:
same on StratisX @dangershony